src/main/kotlin/minesweeper/domain/cell/Empty.kt

```diff
@@ -1,6 +1,9 @@
 package minesweeper.domain.cell
 
-class Empty(position: Position, private var _numberOfNearbyMines: Int = 0) : Cell(position) {
+class Empty(
+    position: Position,
+    private var _numberOfNearbyMines: Int = 0
+) : Cell(position) {
 
     val numberOfNearbyMines get() = _numberOfNearbyMines
```

Comment:
```suggestion
class Empty(
    position: Position,
    numberOfNearbyMines : Int
) : Cell(position) {

    var numberOfNearbyMines: Int = numberOfNearbyMines
    private set
```

backingProperty가 클라이언트에게 노출 된다면 이름 지정 할 때 사용이 힘들거라고 생각해요.

Empty(position, _numberOfNearbyMines = 1) 말이죠. private set을 통하여 표현해주는게 좋지 않을까요? 🤔